docs/production/postgresql-support-table.md

```diff
@@ -8,3 +8,4 @@
 | 8.x                  | 12, 13, 14, 15, 16               |
 | 9.x                  | 12, 13, 14, 15, 16               |
 | 10.x                 | 13, 14, 15, 16                   |
```

Comment:
After backporting, we’ll want to update the 10.x row in both the `10.x` and `main` branches.

Reply:
Done in the 10.2 backport, since ReadTheDocs only rebuilds /stable/ docs on releases.